controllers/flinkcluster_observer.go

```diff
@@ -272,3 +316,15 @@ func (observer *_ClusterStateObserver) observeJob(
 		},
 		observedJob)
 }
+
+func (observer *_ClusterStateObserver) observeJobPods(observedJobPod *corev1.PodList) error {
+	var clusterName = observer.request.Name
+	var jobName = getJobName(observer.request.Name)
+	var inNamespace = client.InNamespace(observer.request.Namespace)
+	var matchingLabels client.MatchingLabels = map[string]string{
+		"app":      "flink",
+		"cluster":  clusterName,
+		"job-name": jobName,
+	}
+	return observer.k8sClient.List(context.Background(), observedJobPod, inNamespace, matchingLabels)
```

Comment:
Can we make this a variable just so that it's more readable?

Reply:
Done